app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -157,6 +158,55 @@ public function addMessageWhenCapture($observer)
         }
     }
 
+    /**
+     * Updates the Bolt transaction status on order status change.
+     *
```

Comment:
Add comment like "note this transaction status is not necessarily same as order status in Bolt server. bolt_transaction_status field keeps track of payment status from magento plugin perspective."